src/main/java/org/jacp/controller/QuestionController.java

```diff
@@ -31,4 +31,11 @@ public ResponseEntity<List<QuestionDto>> getQuestionFilter(@RequestBody SearchDt
         List<QuestionDto> questionDto = mapper.toListQuestionDto(questionEntity);
         return ResponseEntity.ok(questionDto);
     }
+
+    @PostMapping("/getTasksList")
+    public ResponseEntity<List<QuestionDto>> getTasksCurrentCompetition(@RequestBody List<Long> questionIds){
```

Comment:
Can the method name be just "getQuestionsByIds"? 
And above method can be renamed to getQuestionsByFilter. 
And one more thing that the method can be GET mapping anyway, because it's the same as an existing get() method, so it can be similar like:
`
    @GetMapping
    public List<Question> getQuestionsByIds(@RequestParam("ids") List<Long> ids) {
        return questionService.getQuestionsByIds(ids);
    }
`
If you don't think that you will pass in the path more than thousands of values, it can be used like this.

Reply:
Fixed